### PR TITLE
Filter projects

### PR DESCRIPTION
**Pull request by biglovisa**
### What

- Add country/type filtering for Projects

### Why

- To make it easier to only retrieve the Projects that match your criteria.

### SDK Release Checklist

- [x] Have you added an integration test for the changes?
- [x] Have you built the library locally and made queries against it successfully?
- [x] Did you update the changelog?
- [x] Did you bump the package version?
- [ ] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: patch_api/api/estimates_api.py [hunk 1]
```diff
@@ -40,6 +40,8 @@ class EstimatesApi(object):
         "model",
         "make",
         "year",
+        "type",
+        "country",
```
Comment (thdaraujo) [inline patch_api/api/estimates_api.py:44]: 🤔  should this here too?
is the generator adding this filter parameter to all the endpoints or just projects?

**Review by thdaraujo — COMMENTED** [review]
I think the filter is getting added to all the endpoints, no?

File: patch_api/api/estimates_api.py (hunk 1 above)
Comment (biglovisa) [reply patch_api/api/estimates_api.py]: Updated!

**Review by thdaraujo — APPROVED** [review]
🙌

**Pull request merged**